### ISSUE TITLE
Keras dataset shuffle and batch

### ISSUE THREAD

**Issue by davidfox87** [davidfox87/mnist-tfjob-katib#9]
def mnist_dataset(batch_size):
        (x_train, y_train), _ = tf.keras.datasets.mnist.load_data()
        x_train = x_train / np.float32(255)
        y_train = y_train.astype(np.int64)
        train_dataset = (
            tf.data.Dataset.from_tensor_slices((x_train, y_train))
            .shuffle(60000)
            .repeat()
            .batch(batch_size)
        )
        return train_dataset
